src/EPPlusTest/Drawing/DigitalSignatures/DigitalSignatureTests.cs

```diff
@@ -36,7 +38,15 @@ X509Certificate2 GetSelfCert()
             return newCert;
         }
 
-        [ClassInitialize]
+        //Alternate method for if you have a valid cert locally already
```

Comment:
Remove?

Reply:
Nah. Simplifies for us if you don't want to go into each thing and click "trust" if we need to go through valid files some time.